### PR TITLE
Feature/messages

### PR DESCRIPTION
**Pull request by mbarrenechea**
This PR adds:
- Messages tab inside the user profile
- Possibility of sending messages only to investigators that have a user associated
- Icons updated
- Add message button in investigator page
- "Not read" notifications on the messages tab and on each conversation

### CODE REVIEW

**Comment by mbarrenechea** [comment]
Should I wait to the next improvements or do you want merge it now?

**Comment by davidsingal** [comment]
I'm going to merge it, we can open a new PR with improvements

File: config/locales/en.yml [hunk 1]
```diff
@@ -56,6 +56,7 @@ en:
   login: "Login"
   logout: "Log out"
   map: "Map"
+  massage: "Massage"
```
Comment (mbarrenechea) [inline config/locales/en.yml]: Massage?

Comment (davidsingal) [inline config/locales/en.yml]: We have to review these translations, currently "massage" work is not used

**Pull request merged**